lib/routes/objects.js

```diff
@@ -144,6 +144,54 @@ module.exports =
         }
     });
 
+    fastify.route({
+      method: 'GET',
+      url: '/head/:name',
+      schema: {
+        params: {
+          type: 'object',
+          properties: {
+            name: { type: 'string' }
+          }
+        },
+        response: {
+          200: {
+            type: 'array',
+            items: {
+              type: 'object',
+              properties: {
+                enviroment: { type: 'string' },
+                headVersion: { type: ['string', 'null'] },
+                latestVersion: { type: 'string' }
+              }
+            }
+          }
+        }
+      },
+      preHandler: fastify.auth([fastify.verifyAuthentication]),
+      handler:
+        /**
+         * @type {FastifyHandler}
+         */
+        async (req, res) => {
+          const {
+            params: { name }
+          } = req;
+          const envs = await fastify.getEnvs({ name });
+          const heads = await Promise.all(
+            envs.map(async ({ env }) => {
+              const obj = await fastify.getObject({ name, env });
+              if (!obj) return null;
+              const { headVersion, latestVersion } = obj;
+              return { enviroment: env, headVersion, latestVersion };
```

Comment:
```suggestion
              return { environment: env, headVersion, latestVersion };
```

Reply:
I see a bunch more instances of this. @JacopoDaeli you'll want to do a find/replace all